Specs/Renderer/ShaderProgramSpec.js

```diff
@@ -355,6 +358,27 @@ defineSuite([
         expect(renderFragment(context, sp)).toEqual([255, 255, 255, 255]);
     });
 
+    it('creates duplicate uniforms if precision of uniforms in vertex and fragment shader do not match', function() {
```

Comment:
Let's make a similar test using a built-in uniform like [czm_sunDirectionWC](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/AutomaticUniforms.js#L1172).  This is the uniform that caused the original issue (used in the `SkyAtmosphere` vertex and fragment shaders).